DuckDuckGo/Base.lproj/Tab.storyboard

```diff
@@ -28,11 +28,15 @@
                         <rect key="frame" x="0.0" y="0.0" width="768" height="1024"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
```

Comment:
To simplify the review:
- I’ve added a vertical stack view and pinned its edges to the superview.
-I moved the webViewContainer within the stackview.